pkg/cvo/availableupdates.go

```diff
@@ -118,6 +118,13 @@ func calculateAvailableUpdatesStatus(clusterID, upstream, channel, version strin
 		}
 	}
 
+	if len(channel) == 0 {
+		return nil, configv1.ClusterOperatorStatusCondition{
+			Type: configv1.RetrievedUpdates, Status: configv1.ConditionFalse, Reason: "NoChannel",
+			Message: "The update channel has not been assigned.",
```

Comment:
nit: "assigned" -> "configured"?  To make it more clear that this is up to the admin to fix this.